frontend/src/features/admin-form/create/common/CreatePageSidebar/CreatePageSidebar.tsx

```diff
@@ -1,10 +1,12 @@
 import { useCallback } from 'react'
-import { BiGitMerge } from 'react-icons/bi'
-import { Stack } from '@chakra-ui/react'
+import { BiGitMerge, BiQuestionMark } from 'react-icons/bi'
+import { Stack, Tooltip } from '@chakra-ui/react'
```

Comment:
```suggestion
import { Stack } from '@chakra-ui/react'
import Tooltip from '~components/Tooltip'
```

Reply:
thank you!! [24ab287](https://github.com/opengovsg/FormSG/pull/4960/commits/24ab2876e7c21e1dc239b6a2b3d2aebc012eb919)